input/fsh/CGGeneral.fsh

```diff
@@ -47,6 +47,8 @@ One important note is that Annotation is a FHIR data type, this is **NOT** about
 * component ^slicing.discriminator.path = "code"
 * component ^slicing.rules = #open
 * component ^slicing.description = "Slice based on the component.code pattern"
+* component.extension contains RelatedArtifactComponent named related-artifact 0..*
```

Comment:
@rhdolin - why did you move this to GenomicBase?

Reply:
I'll revert this one.